Makefile

```diff
@@ -120,9 +120,9 @@ run-apiserver:
 
 run-build:
 	GOGC=1000 go install -v \
-		-ldflags '-X main.version=$(if $(BUILD_VERSION),$(BUILD_VERSION),devbuild)' \
-		./volcli/volcli/ ./volplugin/volplugin/ ./apiserver/apiserver/ ./volsupervisor/volsupervisor/ ./volmigrate/volmigrate/
-	cp $(GUESTBINPATH)/* bin/
+		 -ldflags '-X main.version=$(if ${BUILD_VERSION},${BUILD_VERSION},devbuild)' \
```

Comment:
oh right. I fixed this as a result of a merge confict I had after we merged your branch. I will resolve this in a minute.